src/btop_config.hpp

```diff
@@ -53,6 +53,7 @@ namespace Config {
 #ifdef GPU_SUPPORT
 	const vector<string> show_gpu_values = { "Auto", "On", "Off" };
 #endif
+    const vector<string> base_10_bitrate_values = { "True", "False", "Auto" };
```

Comment:
Please make this the first option as well.

```suggestion
    const vector<string> base_10_bitrate_values = { "Auto", "True", "False" };
```